pkg/docker/compose/config/schema.go

```diff
@@ -224,10 +206,10 @@ type port struct {
 }
 
 func (p *port) Decode(into mapdecode.Into) error {
-	intVal := 0
-	err := into(&intVal)
+	var int64Val int64
```

Comment:
Changed from `int` to `int64` since we don't want the behaviour to change from one platform to the next (int could be 32 bits in theory).